LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
Can we please use LGPLv3 ?  (or Apache v2.0) is also fine.

As this is not directly linking the code of glusterfs, it doesn't bind to have a GPLv2 licence. Lets make it more adaptive in other echo systems, and keep Apache v2.0 IMO.

Reply:
LGPLv3 looks good to me @amarts, however moving to Apache v2.0 looks a little tricky as the project is originally open sourced under LGPLv2 moving on would require us to communicate the same to all contributors

Reply:
One such example is: https://github.com/gluster/glustercli-python/issues/28

Reply:
Ack, lets go with LGPLv3, which is compatible with Apachev2.0

Reply:
Done @amarts